wee_slack.py

```diff
@@ -4036,6 +4051,31 @@ def command_thread(data, current_buffer, args):
 command_thread.completion = '%(threads)'
 
 
+@slack_buffer_required
+@utf8_decode
+def command_subscribe(data, current_buffer, args):
+    """
+    /slack subscribe <thread>
+    Subscribe to a thread, so that you are alerted to new messages.  When in a
+    thread buffer, you can omit the thread id.
+    """
+    channel = EVENTROUTER.weechat_controller.buffers[current_buffer]
+
+    if not args:
+        w.prnt('', 'Usage: /slack subscribe <thread>')
+        return w.WEECHAT_RC_ERROR
+
+    if args:
+        msg = get_msg_from_id(channel, args)
+
+    if not msg and isinstance(channel, SlackThreadChannel):
+        msg = channel.parent_message
+
+    # TODO: Looks like there is no RTM api for this?
```

Comment:
Yeah, I couldn't find any either. I tried the api the web client uses, but that just says `not_allowed_token_type` if I try it with the tokens we use for wee-slack.

Reply:
This now works if one has a special token.